impl/src/main/java/com/sun/faces/context/BaseContextMap.java

```diff
@@ -16,14 +16,7 @@
 
 package com.sun.faces.context;
 
-import java.util.AbstractCollection;
-import java.util.AbstractMap;
-import java.util.AbstractSet;
-import java.util.Collection;
-import java.util.Enumeration;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can you please reconfigure your IDE to not do that anymore? Wildcard imports are highly discouraged.